company/update.js

```diff
@@ -0,0 +1,88 @@
+import { failure, success, validationError } from '../libs/response-lib';
+import * as dynamoDbLib from '../libs/dynamodb-lib';
+import { companySchema } from '../validation/companySchema';
+import { flattenObject } from '../libs/utils-lib';
+
+export const main = async event => {
+  /** @type {string} companyId - UUID */
+  const companyId = event.pathParameters.id;
+
+  /** @type {import('../typings/company).Company} */
+  const data = JSON.parse(event.body);
+
+  let value = null;
+
+  try {
+    value = await companySchema('optional').validateAsync(data);
+  } catch (e) {
+    const errorMessages = e.details.map(detail => detail.message);
+    return validationError(errorMessages);
+  }
+
+  const params = {
+    TableName: process.env.COMPANIES_TABLE_NAME,
+    Key: {
+      companyId
+    },
+    // 'UpdateExpression' defines the attributes to be updated
+    // 'ExpressionAttributeValues' defines the value in the update expression
```

Comment:
comments seem like lefties from the copypaste

Reply:
Actually, I intentionally left them. Because the update is a bit different, and whoever is reading the code will understand the two fields. If you think they are not needed, I can remove them

Reply:
Got it 🙃 I'm fully comfortable to keep those 🙃